Dockerfile.aro

```diff
@@ -7,6 +7,7 @@ WORKDIR ${GOPATH}/src/github.com/openshift/ARO-Installer
 USER root
 RUN yum update -y
 COPY . ${GOPATH}/src/github.com/openshift/ARO-Installer/
+RUN git config --system --add safe.directory '*'
```

Comment:
Seems fine especially because this is the recommended fix from other teams. Wondering about the use of wildcard (*) here but likely makes sense as the underlying paths are a bit obfuscated